app/assets/javascripts/get_location.js.erb

```diff
@@ -0,0 +1,18 @@
+function initAutocomplete() {
```

Comment:
Had to convert this to a .js.erb file so that I could pull out the Rails environment variable!